Exiled.API/Features/Player.cs

```diff
@@ -908,7 +908,16 @@ public int Scp330Usages
         /// Gets player's items.
         /// </summary>
         public Inventory.SyncListItemInfo Items => Inventory.items;
-
+        
+        /// <summary>
+        /// Gets or sets a value indicating whether or not the player can send inputs.
+        /// </summary>
+        public bool NetworkforceStopInputs
```

Comment:
This name's kinda gross. Maybe go with something more descriptive like `CanSendInputs`.

Reply:
Or `CanMove`

Reply:
Yeah I think you're right. `CanSendInputs` is fine.